<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retail Store Management System v1.4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="lookupStockButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Lookup Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStockButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateStockButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftTopLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Manage Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customerInvoiceButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer invoice</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightTopLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Sell Item/Manage Invoice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellItemButton">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>110</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sell Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteInvoiceButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Invoice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit Program</string>
    </property>
   </widget>
   <widget class="QListWidget" name="categoryList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Accessories</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cell Phone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Electronics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laptop Computers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miscellaneous</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monitors &amp; TV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Smart Watch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sports</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="brandInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="brandLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>item Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Placeholder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="categoryLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Retail Price</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="amountInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="priceInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="amountInput_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="categoryLabel_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabel_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>item Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="brandLabel_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Name</string>
    </property>
   </widget>
   <widget class="QListWidget" name="categoryList_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Accessories</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cell Phone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Electronics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laptop Computers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miscellaneous</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monitors &amp; TV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Smart Watch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sports</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="brandInput_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemInput_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Placeholder</string>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>541</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice #</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceInput">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="signoutButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>841</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="wholesaleInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="wholesaleLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wholesale Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cartButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteStockButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Stock</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="shipmentInvoiceButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Shipment Invoice</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentCartSize">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>520</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
